src/main/java/core/basesyntax/TextFiles.java

```diff
@@ -0,0 +1,168 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.attribute.FileTime;
+import java.util.List;
+import java.util.Scanner;
+
+public class TextFiles {
+    private static Scanner input = new Scanner(System.in);
+    private static String INPUT = input.nextLine();
+    private static String[] COMMANDS = {"create", "read", "info", "help", "helpC", "exit"};
+    private static int CASE_NUM = 0;
+    private static String[] DESCRIPTION = {" - create a new file or overrides the existing one",
+            " - output the file`s text",
+            " - output file`s info",
+            " - output commands info",
+            " - output specific command info",
+            " - output info about the specific command"};
+
+    public static void main(String[] args) throws IOException {
+        System.out.println("Please enter your command");
+        do {
+            commandExecute();
+        } while (!INPUT.equals(COMMANDS[COMMANDS.length - 1]));
+    }
+
+    public static void checkInput() {
+        String[] args = INPUT.split(" ");
+        if (args.length == 1) {
+            if (INPUT.equals(COMMANDS[3])) {
+                CASE_NUM = 4;
+            } else if (INPUT.equals(COMMANDS[5])) {
+                CASE_NUM = 6;
+            }
+        }
+        if (args.length == 2) {
+            for (String s : COMMANDS) {
+                if (args[0].equals(COMMANDS[4]) && args[1].equals(s)) {
+                    CASE_NUM = 5;
+                    break;
+                }
+            }
+        }
+        if (args.length == 3) {
+            for (int i = 0; i < COMMANDS.length; i++) {
+                //"^([\\w\\s+\\-/]|(\\\\\\\\))+$"
+                if (args[0].equals(COMMANDS[i]) && args[1].matches("^([a-zA-Z0-9 +_/-]|\\\\\\\\)+")
+                        && args[2].matches("^([a-zA-Z0-9 +_/-]|\\\\\\\\)+")) {
+                    CASE_NUM = i + 1;
+                }
+            }
+        }
+    }
+
+    public static void commandExecute() throws IOException {
+        String[] args = INPUT.split(" ");
+        checkInput();
+        switch (CASE_NUM) {
```

Comment:
can we check ars[0] whether it's equal to any element form COMMANDS ? this way you can get rid of checkInput() method